lib/fastlane/plugin/revenuecat_internal/actions/git_clone_and_push.rb

```diff
@@ -0,0 +1,40 @@
+require 'fastlane/action'
+require 'fastlane_core/configuration/config_item'
+require_relative '../helper/revenuecat_internal_helper'
+require_relative '../helper/versioning_helper'
+
+module Fastlane
+  module Actions
+    class GitCloneAndPushAction < Action
```

Comment:
Might be good to add some tests, even if there is basically no logic. Mostly to make sure the parameters and such don't change without us knowing.

Reply:
Done!